### DISCUSSION TITLE
2022-04-11 Pyaerocom meeting

### DISCUSSION THREAD

**Discussion by lewisblake** [metno/pyaerocom#623] [Development meetings]
## Follow up from last meeting

Apr 4th #618

## Issues & Discussions

- [ ] Is pip still broken? Is there an expected fix?
- [ ] Documentation for pyaerocom development
- Creation of internal documentation (wiki) and manuscript. Etter påskefeire?
- Inspiration Reference:
  - ESMVALTOOL
  https://www.esmvaltool.org/
  https://gmd.copernicus.org/articles/14/3159/2021/
  - cis tools
  https://www.cis-tools.co.uk/
  https://gmd.copernicus.org/articles/9/3093/2016/
- [ ] Python Parallelization Performance Price: workshop/competition, rules and dates TBA
- [ ] Find a date for a CMIP6 intake catalog Hackathon 

## PRs

- #547 
  - [ ] needs feedback: @dulte 
- #550
  - [ ] missing tests
- #582
  - [ ] fix broken tests
- #593
  - [x] what is needed to finish?
- #602
  - [ ] needs feedback and conversation vs typer-based CLI: @avaldebe 

## For next

- [ ] April release
- [X] Give Lewis ability to create "Development meeting" discussions :upside_down_face:

**Comment by avaldebe** [comment]
@lewisblake @dulte @charlienegri are now maintainer

**Comment by avaldebe** [comment]
> Is pip still broken?

@lewisblake what to you mean?

Is this about failing CI? or failed Proj8 pip install on PPI/REL8?
The recent CI failures seem to be related to bad merges and bad formatting.
pip install on machines with Proj8 is addressed by #620.

**Reply by lewisblake** [reply]
This was related to pip install issues Willem was having last week. Conda install sufficed for this use case. Btw, we never actually had this meeting (not many people around), so many of these issues can be pushed to our next pyaerocom meeting.
